src/Network/Network/ApplicationGateway/NewAzureApplicationGatewayCommand.cs

```diff
@@ -121,6 +127,12 @@ public class NewAzureApplicationGatewayCommand : ApplicationGatewayBaseCmdlet
              HelpMessage = "The list of backend http settings")]
         public PSApplicationGatewayBackendHttpSettings[] BackendHttpSettingsCollection { get; set; }
 
+        [Parameter(
+             Mandatory = false,
+             ValueFromPipelineByPropertyName = true,
+             HelpMessage = "The list of ssl profiles")]
+        public PSApplicationGatewaySslProfile[] SslProfiles { get; set; }
```

Comment:
please rename SslProfiles as SslProfile
powershell use singularity for parameter names:
https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/design-guidelines/cmdlet-best-practices.md

Reply:
Same reason as above.